crates/holochain/src/local_network_tests.rs

```diff
@@ -73,6 +81,9 @@ async fn conductors_call_remote(num_conductors: usize) {
             }
         })
         .await;
+
+    // Ensure that all the create requests were received and published.
+    consistency_60s(cells.iter()).await;
```

Comment:
Test without an assertion at the end, this is where the flaky behaviour is allowed to creep in. When this is added we see that the test isn't entirely reliable on any platform.